### ISSUE TITLE
Accessibility : underline links that are in text content

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-server#5074]
Issue reported by the Accessibility Foundation (see #4640 ) :

"1.4.1 Use of Color
On the homepage (https://nl.openfoodfacts.org/) under the heading "Donate to support
our work" two links are visible that can be seen in a different color. People with color
blindness cannot perceive these links. We recommend that you provide the links with an
underline so that the links are distinguished from the normal text."

This comment is valid for text links that are inside other textual content. e.g. "some words [some words with a link] other words".
For instance in the "Discover" page: https://world.openfoodfacts.org/discover
- there is a link in the sentence "We need your donations to fund the Open Food Facts 2021 budget and to continue to develop the project."
- and below there are links in sentences like "Check-out the cool reuses or make your own!".

One solution would be to change the CSS so that links that are inside a <p> paragraph are underlined.

New CSS styles can be added to scss/_off.scss
after updating the SCSS files, "npm run build" needs to be re-run to create the CSS files.

**Comment by hangy** [comment]
I think this was fixed with the design overhaul

**Comment by alexgarel** [comment]
Indeed. Thanks @hangy
